es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,258 +1,258 @@
 
 <br>
 
 **4. Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
 
-&#10230;
+&#10230; Matriz de confusión ― La matriz de confusión es usada para tener un mejor panorama cuando se evalua el rendimiento del modelo. Se define como:
 
 <br>
 
 **5. [Predicted class, Actual class]**
 
-&#10230;
+&#10230; [Clase Predicha, Clase Real]
 
 <br>
 
 **6. Main metrics ― The following metrics are commonly used to assess the performance of classification models:**
 
-&#10230;
-
+&#10230; Métricas principales ― Las siguientes métricas son comúnmente usadas para evaluar el desempeño de modelos clasificatorios
+ 
 <br>
 
 **7. [Metric, Formula, Interpretation]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Interpretación]*
 
 <br>
 
 **8. Overall performance of model**
 
-&#10230;
+&#10230; Desempeño en general del modelo
 
 <br>
 
 **9. How accurate the positive predictions are**
 
-&#10230;
+&#10230; ¿Qué tan precisan son las predicciones positivas?
 
 <br>
 
 **10. Coverage of actual positive sample**
 
-&#10230;
+&#10230; Cobertura de la muestra positiva
 
 <br>
 
 **11. Coverage of actual negative sample**
 
-&#10230;
+&#10230; Cobertura de la muestra negativa
 
 <br>
 
 **12. Hybrid metric useful for unbalanced classes**
 
-&#10230;
+&#10230; Métrica híbrida útil para clases no balanceadas
 
 <br>
 
 **13. ROC ― The receiver operating curve, also noted ROC, is the plot of TPR versus FPR by varying the threshold. These metrics are are summed up in the table below:**
 
-&#10230;
+&#10230; Curva de operación del receptor (ROC, por sus siglás en Inglés) ― La curva receptora de operaciones, por sus siglás en inglés ROC, es la gráfica de los Tasa de verdaderos positivos (TPR) contra los Tasa de falsos positivos (FPR) variando el umbral. Estas métricas son resumidas en la tabla siguiente: 
 
 <br>
 
 **14. [Metric, Formula, Equivalent]**
 
-&#10230;
+&#10230; [Métrica, Fórmula, Equivalente]
 
 <br>
 
 **15. AUC ― The area under the receiving operating curve, also noted AUC or AUROC, is the area below the ROC as shown in the following figure:**
 
-&#10230;
+&#10230; AUC ― El área bajo la curva ROC, también llamda AUC o AUROC, es la área bajo de la curva ROC como se muestra en la siguiente figura:
 
 <br>
 
 **16. [Actual, Predicted]**
 
-&#10230;
+&#10230; [Valor actual, Predicho]
 
 <br>
 
 **17. Basic metrics ― Given a regression model f, the following metrics are commonly used to assess the performance of the model:**
 
-&#10230;
+&#10230; Métricas básicas ― Dado un modelo de regresión f, las siguientes métricas son comúnmente usadas para evaluar el desempeño del modelo:
 
 <br>
 
 **18. [Total sum of squares, Explained sum of squares, Residual sum of squares]**
 
-&#10230;
+&#10230; [Suma de cuadrados explicada, Explicación de la suma de cuadrados, Suma de los cuadrados de los residuales]
```

Comment:
Something went wrong here haha. It is:
_Suma de cuadrados total, Suma de cuadrados explicada, Suma de los cuadrados de los residuales]_